tdrs-backend/tdpservice/parsers/aggregates.py

```diff
@@ -1,9 +1,10 @@
 """Aggregate methods for the parsers."""
 from .row_schema import SchemaManager
-from .models import ParserError
+from .models import ParserError, ParserErrorCategoryChoices
 from .util import month_to_int, \
     transform_to_months, fiscal_to_calendar, get_prog_from_section
 from .schema_defs.utils import get_program_models, get_text_from_df
+from django.db.models import Q
```

Comment:
```suggestion
from django.db.models import Q as Query
```

I've definitely done this in the past but was curious if you thought a renaming here might make the code lower down more intuitive.

Reply:
Yes I like that, will make the change!